### ISSUE TITLE
def file generation

### ISSUE THREAD

**Issue by aforgio** [enthought/pyql#187]
Hi everybody,

I'm new in Python and in QuantLib too. I would to install PyQL on a Windows machine, so I have set up the boost  using Visual Studio and I have done the first built of the QL project and I've also patched the generate_symbols.py file. Please, can someone tell me how to generate a def file (and what is it)? 

Thank you very much

**Comment by happynothing** [comment]
I think you can't do that! because the script uses "nm" command which is only in linux.
I don't know how they can pass the test!

**Comment by yunjianyang** [comment]
Exactly.... Finally I figured out there is no "nm" on windows... No idea how they pass the test...
